jaggr-core/WebContent/dojo/_loaderExt.js

```diff
@@ -280,6 +288,22 @@ combo.isI18nResource = combo.isI18nResource || function(mid) {
 	}
 };
 
+combo.addBootLayerDeps = function(deps) {
+	excludes = excludes.concat(deps);
+};
+
+/*
+ * Returns true if the specified module is defined, false otherwise
+ */
+combo.isDefined = function(name) {
+	try {
+		require(name);
+		return true;
+	} catch (ignore) {
+	}
+	return false;

```

Comment:
Actually, from a stylistic point of view, I don't like having functions without a top level return statement.  As you say, functionally, it makes no difference, but I'd prefer to leave it as is.